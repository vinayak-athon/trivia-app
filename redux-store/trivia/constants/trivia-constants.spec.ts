import { TriviaActionConsts } from ".";

describe("TriviaActionConsts", () => {
  it("should have to Match Snapshot", () => {
    expect(TriviaActionConsts).toMatchSnapshot();
  });
});
