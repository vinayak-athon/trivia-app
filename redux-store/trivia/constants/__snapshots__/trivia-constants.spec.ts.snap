// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`TriviaActionConsts should have to Match Snapshot 1`] = `
Object {
  "GET_TRIVIA_QUESTIONS_FAILURE": "GET_TRIVIA_QUESTIONS_FAILURE",
  "GET_TRIVIA_QUESTIONS_FETCHING": "GET_TRIVIA_QUESTIONS_FETCHING",
  "GET_TRIVIA_QUESTIONS_SUCCESS": "GET_TRIVIA_QUESTIONS_SUCCESS",
  "TRIVIA_QUESTIONS_RESET": "TRIVIA_QUESTIONS_RESET",
}
`;
